data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -1074,10 +1075,16 @@ craft-beekeeper                                     # icon z17- (also has captio
 craft-blacksmith                                    # icon z17- (also has caption(optional) z18-)
 craft-brewery                                       # icon z17- (also has caption(optional) z18-)
 craft-carpenter                                     # icon z17- (also has caption(optional) z18-)
+craft-caterer                                       # icon z17- (also has caption(optional) z18-)
 craft-confectionery                                 # icon z17- (also has caption(optional) z18-)
 craft-electrician                                   # icon z17- (also has caption(optional) z18-)
 craft-electronics_repair                            # icon z17- (also has caption(optional) z18-)
 craft-gardener                                      # icon z17- (also has caption(optional) z18-)
+craft-grinding_mill                                 # icon z17- (also has caption(optional) z18-)
+craft-handicraft                                    # icon z17- (also has caption(optional) z18-)
+craft-hvac                                          # icon z17- (also has caption(optional) z18-)
+craft-key_cutter                                    # icon z17- (also has caption(optional) z18-)
+craft-locksmith                                     # icon z17- (also has caption(optional) z18-)
```

Comment:
I don't think those crafts are important enough to put them above e.g. `amenity-school`.
I know its not consistent ATM.. I think most other crafts (and many shops) should be demoted too..

Reply:
Moved back to where they were originally